terraform/cp4ba/workspace.tmpl.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Update this to contain all of the variables.  It still refers to Multicloud management and is missing all the `portworx` and `cp4ba` variables.